src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,246 @@
+                    System.out.println("You can't do something that doesn't exist, or can you?");
+                }
+                System.out.println(separatingLine);
+            } else if(inputStatement.startsWith("delete")){
+                System.out.println(separatingLine);
+                try {
+                    deleteItem(inputStatement, listOfItems);
+                    listWordCount = listWordCount - 1;
+                    System.out.println("Now you have " + (listWordCount) + " tasks in the list.");
+                }catch(IndexOutOfBoundsException|NumberFormatException e){
+                    System.out.println("You can't delete that!");
+                }
+                System.out.println(separatingLine);
+            }else if(inputStatement.startsWith("todo")|inputStatement.startsWith("event")
+                    |inputStatement.startsWith("deadline")) {
+                //proceeds to add task if task is valid.
+                try {
+                    listWordCount = classifyAndAddTask(inputStatement, separatingLine, listOfItems, listWordCount);
+                }catch (StringIndexOutOfBoundsException | IllegalCommandsException e){
+                    //throws exception when adding invalid tasks.
+                    printBetweenLines(separatingLine, "That's not a valid task!");
+                }
+
+            } else{
+                //throws exception when inputting invalid commands.
+                printBetweenLines(separatingLine, "That's not a valid command!");
+            }
+            //update the text document
+            try {
+                writeToFile(defaultPath, listOfItems);
+            } catch (IOException e) {
+                System.out.println("Something went wrong: " + e.getMessage());
+            }
+
+
+        }
+        //ending messages
+        printBetweenLines(separatingLine, "We shall meet again...");
+    }
+
+    private static void printWelcomeText(String separatingLine) {
+        System.out.println(separatingLine);
+        System.out.println("Hey, welcome to my version of Duke and please make good use of it.");
+        System.out.println("The bot is meant to be a bit cheeky at times so don't mind it.");
+        System.out.println("Here is a list of commands you can try out:");
+        System.out.println("\u2022[todo ___] adds a todo task to your list.");
+        System.out.println("\u2022[deadline ___ /by ___] adds a deadline task to your list with a deadline.");
+        System.out.println("\u2022[event ___ /at ___] adds a event task to your list with a duration.");
+        System.out.println("\u2022[done (index of task)] marks a task as done in your list.");
+        System.out.println("\u2022[list] prints out your current list and status of tasks.");
+        System.out.println("\u2022[bye] exits the programme.");
+    }
+
+    private static int classifyAndAddTask(String inputStatement, String separatingLine
```

Comment:
Maybe you should improve the abstraction here e.g: methods for classify and add tasks.